user_docs/en/changes.md

```diff
@@ -13,6 +13,7 @@
 
 * Component updates:
   * Updated Unicode CLDR to version 45.0. (#16507, @OzancanKaratas)
+  * Updated fast_diff_match_patch (used to detect changes in terminals and other dynamic content) to version 2.1.0. (#16508, @codeofdusk)

```

Comment:
Not sure, but this does not seem to be an update but rather standard diff match patch being replaced by fast diff match patch.
@codeofdusk, is it correct? If yes, the change log item should be modified.